### DISCUSSION TITLE
Can you whitelist a local server (localhost) when using MAMP on Mac?

### DISCUSSION THREAD

**Discussion by Chacapamac** [uBlockOrigin/uBlock-issues#2333] [General] [answered]
I’m a web developer and I have multiple websites in development of for testing on my Mac using MAMP PRO Server. I like to whitelist all these websites from UBlock. Possible?

**Comment by ghost** [comment, marked as answer]
If you can match them in one of these ways: https://github.com/gorhill/uBlock/wiki/How-to-mark-a-web-site-as-trusted

**Comment by Chacapamac** [comment]
Thank You, whoa! great app my friend.
http://localhost/*  Work Perfectly.
I just ditch Chrome as my main browser as Chrome manifest v3 will force you to watch their F*** Ads and probably even reduce the ability of blocking add plugins — My main browser is now my Firefox Developper Edition and I was searching for a trusted Add Block plugin. I find it with UBLOCK — Working Great! Thank You!

Just give a 5 stars rating!
